README.md

```diff
@@ -50,142 +43,8 @@ var dto = mapper.CarToCarDto(car);
 dto.NumberOfSeats.Should().Be(10);
 ```
 
-## Configuration
+[Read the docs](https://riok.github.io/mapperly/docs/getting-started/installation) for any further information.
```

Comment:
In addition to this, I would add an additional main title + paragraph for the documentation, so it is easier to find.

Reply:
There is already (just before the quickstart).